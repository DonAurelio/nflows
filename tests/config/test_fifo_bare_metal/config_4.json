{
    "dag_file": "./tests/workflows/test_fifo_bare_metal/config_4.dot",
    "scheduler_type": "fifo",
    "mapper_type": "bare_metal",
    "mapper_mem_policy_type": "default",

    "core_avail_mask": "0x01000001",
    "flops_per_cycle": 32,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1000000000,

    "distance_matrices": {
        "latency": "./tests/system/test_fifo_bare_metal/4_lat.txt",
        "bandwidth": "./tests/system/test_fifo_bare_metal/4_bw.txt"
    },

    "log_base_name": "./tests/output/test_fifo_bare_metal/config_4",
    "log_date_format": ""
}
