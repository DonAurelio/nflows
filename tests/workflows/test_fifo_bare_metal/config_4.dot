digraph DataRedistribution {
    root    [size=2]; // Ignored in processing.
    end     [size=2]; // Ignored in processing.

    Task_1  [size=1000000000]; // 1Gflop
    Task_2  [size=1000000000];
    Task_3  [size=1000000000];

    root -> Task_1  [size=2]; // Edge ignored.
    root -> Task_2  [size=2]; // Edge ignored.
    
    Task_1 -> Task_3  [size=4000000000];
    Task_2 -> Task_3  [size=8000000000];

    Task_3 -> end   [size=2]; // Edge ignored.
}
