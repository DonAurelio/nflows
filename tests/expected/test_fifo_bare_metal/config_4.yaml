trace:
  name_to_thread_locality:
    Task_1: {numa_id: 0, core_id: 0}
    Task_2: {numa_id: 1, core_id: 24}
    Task_3: {numa_id: 1, core_id: 24}
